irohad/main/impl/pg_connection_init.cpp

```diff
@@ -279,15 +279,28 @@ CREATE TABLE IF NOT EXISTS account_has_grantable_permissions (
       + R"() NOT NULL,
     PRIMARY KEY (permittee_account_id, account_id)
 );
-CREATE TABLE IF NOT EXISTS position_by_hash (
-    hash varchar unique not null,
+CREATE TABLE IF NOT EXISTS tx_positions (
+    creator_id varchar(64),
```

Comment:
64 seems not enough to hold any valid account name. please see the regex for it:
https://github.com/hyperledger/iroha/blob/45c9dd48ec40c453d876818f0fcd6078522808d2/shared_model/validators/field_validator.cpp#L74-L97